tests/components/homeassistant/test_scene.py

```diff
@@ -0,0 +1,30 @@
+"""Test Home Assistant scenes."""
+from unittest.mock import patch
+
+from homeassistant.setup import async_setup_component
+
+
+async def test_reload_config_service(hass):
+    """Test the reload config service."""
+    assert await async_setup_component(hass, "scene", {})
+
+    with patch(
+        "homeassistant.config.load_yaml_config_file",
+        autospec=True,
+        return_value={"scene": {"name": "Hallo", "entities": {"light.kitchen": "on"}}},
+    ), patch("homeassistant.config.find_config_file", return_value=""):
+        await hass.services.async_call("scene", "reload", blocking=True)
+        await hass.async_block_till_done()
```

Comment:
Do we need to block till done if we call the service blocking?

Reply:
The `async_add_entities` method passed into platform setup is `EntityPlatform._async_schedule_add_entities`, so it will use `async_create_task` under the hood.